routingpy/routers/mapbox_osrm.py

```diff
@@ -501,6 +504,8 @@ def matrix(
         if fallback_speed:
             params['fallback_speed'] = str(fallback_speed)
 
+        profile = profile.replace('mapbox/','')
```

Comment:
I believe this is superfluous, it's been without the `mapbox/` modifier before as well.